src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -57,6 +57,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Routing" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="2.2.0" />
```

Comment:
I have a slight concern that this may cause a conflict in Functions V3, as they may have changed their internal dependency on this package since they are on .NET Core 3.1.

Reply:
I had the same fear, but then I realized this was the latest version of Kestrel, so they couldn't have a newer dependency than this. :)

Reply:
Good to know! 

If I remember from my previous porting to .NET Core 3.0, they started shipping a lot of these packages as a part of .NET Core, so you no longer explicitly reference the package.

That being said, it is still probably a good idea to do a final end-to-end test with both Functions V2 and Functions V3.

Reply:
I've actually verified both already (Functions 3.0 locally and Functions 2.0 on Azure), so I think we're good.